lib/Mojolicious/Command/Author/generate/secret.pm

```diff
@@ -0,0 +1,82 @@
+package Mojolicious::Command::Author::generate::secret;
+use Mojo::Base 'Mojolicious::Command';
+use Mojo::File qw(path);
+use Mojo::Util qw(urandom_urlsafe);
+
+has description => 'Generate secret';
+has usage       => sub { shift->extract_usage };
+
+sub run {
+  my ($self, $secret_file) = (shift, shift);
+
+  $secret_file //= $self->app->secrets_file;
+
+  my $token = urandom_urlsafe();
+
+  print "Writing @{[ length($token) ]} byte to $secret_file\n";
```

Comment:
What is the intent of this message? It's probably good to inform users that the file is being written, but showing the expected file size may be less informative?

Reply:
I agree, removed the number of bytes